sql/rowexec/rel_iters.go

```diff
@@ -243,35 +243,222 @@ func (i *offsetIter) Close(ctx *sql.Context) error {
 	return i.childIter.Close(ctx)
 }
 
-type jsonTableRowIter struct {
-	colPaths []string
-	schema   sql.Schema
+type jsonTableColOpts struct {
+	name      string
+	typ       sql.Type
+	forOrd    bool
+	exists    bool
+	defErrVal interface{}
+	defEmpVal interface{}
+	errOnErr  bool
+	errOnEmp  bool
+}
+
+// jsonTableCol represents a column in a json table.
+// TODO: this wastes memory for each column
```

Comment:
This was left over from a previous implementation; it had `Opts` as a struct rather than a pointer, taking up extra memory for the `JSONTableCol`s that contain NestedCols. Will remove